### DISCUSSION TITLE
Easiest invulnerable boss to solo clear to for testing?

### DISCUSSION THREAD

**Discussion by ArtOfShred** [DakJaniels/LuiExtended#244] [General]
Anyone have any idea what the best place to do testing for adding an invulnerability highlight for UnitFrames would be? Not familiar with like the last 3 years of bosses added and the only ones I can think of that go invulnerable are bosses in Trials.

If theres's something in a Dungeon I can feasibly solo to on normal difficulty that has an invulnerability mechanic that'd work (even better if there's a World Boss somewhere).

**Comment by vboomx** [comment]
Hi! For sure in High Isle there in Faun Falls WB that have this shown durin indrik wall phase
https://imgur.com/a/G73vCfn

**Reply by vboomx** [reply]
First secret boss in Shipwright regret dungeon have this aswell so its pretty easy to reach

Oblivion portals in Deadlands last boss have this option aswell

**Comment by ArtOfShred** [comment]
Awesome thanks, I'll see if I can take a look at this and implement it before the update comes out on Monday!

**Comment by ArtOfShred** [comment]
Aww gyeah implemented!
I brightened the color up a little bit too but didn't take a screenshot first. I updated Guards to use the same functionality too.
![](http://puu.sh/JT8ox.jpg)

**Reply by saenic** [reply]
Nice.
Can you adjust the text? I for myself would just use the color without the "-Invulnerable-" text.

**Reply by ArtOfShred** [reply]
Yeah I think the - Invulnerable - kinda looks meh. I'll probably adjust that tomorrow. The visual effect on the bar is more then enough of an indicator that the mob is invulnerable.

**Reply by saenic** [reply]
My initial idea was to show a shield icon where the execute skull is. But the bar color is fine as well.
